homeassistant/components/tessie/helpers.py

```diff
@@ -0,0 +1,19 @@
+"""Tessie helper functions."""
+
+from typing import Any
+
+from tesla_fleet_api.exceptions import TeslaFleetError
+
+from homeassistant.exceptions import HomeAssistantError
+
+from . import _LOGGER
+
+
+async def handle_command(command) -> dict[str, Any]:
+    """Handle a command."""
+    try:
+        result = await command
+    except TeslaFleetError as e:
+        raise HomeAssistantError(f"Tessie command failed, {e.message}") from e
```

Comment:
Do we want to add a test for this one?

Reply:
It has coverage in https://github.com/home-assistant/core/pull/120641 from last week but I can add it here too.

Reply:
Aah check, let's review that one and rebase this one after that 👍